docker_images/ci-docs/Dockerfile

```diff
@@ -3,6 +3,5 @@ FROM python:slim
 RUN apt-get update \
     && apt-get install -y enchant-2 git make latexmk texlive-latex-base texlive-fonts-recommended texlive-fonts-extra texlive-latex-extra texlive-xetex xindy graphviz
 
-COPY requirements.txt /tmp/
-
-RUN pip3 install --requirement /tmp/requirements.txt
+# Install the requirements from the conan docs repo so they stay in sync when we regenerate the image
+RUN pip3 install --requirement https://raw.githubusercontent.com/conan-io/docs/master/requirements.txt
```

Comment:
Why getting the requirements from internet is better that getting the local ones?

Reply:
we started caching the requirements because the PR's in the docs repo were taking a lot of time just installing the requirements to build the docs and recently we had some problems because the requirements from the image were out of sync with the ones used from the docs, this is just something to make it a bit less fragile but we have to think something better than this.